src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -13,6 +13,8 @@ public class ServerJob : IIdentifiable
     {
         public int Id { get; set; }
 
+        public string ConnectionFilter { get; set; }
```

Comment:
I think this is fine for now, but if we do more of this we'll have to add "scenario args" because this doesn't scale super well